docker/Dockerfile.rocm

```diff
@@ -37,7 +37,7 @@ ENV SGLANG_SET_CPU_AFFINITY=1
 ENV SGLANG_ALLOW_OVERWRITE_LONGER_CONTEXT_LEN=1
 ENV NCCL_MIN_NCHANNELS=112
 
-ENV MOE_PADDING=1
+ENV MOE_PADDING=0
```

Comment:
We need to keep MOE_PADDING on for performance, to error it incurs we need to fix it.